module-types/Palo Alto/PAN-PA-5400-MPC-A.yaml

```diff
@@ -29,3 +33,6 @@ interfaces:
     type: 10gbase-x-sfpp
     mgmt_only: true
     label: MGT-B
+  - name: USB
```

Comment:
What is the USB interface for? Interfaces is reserved for network interfaces, not for management, file transfer or anything like that